src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -457,11 +498,36 @@ private void release()
             }
         }
 
+        private long challenge(
+            long now,
+            long traceId,
+            long delay,
+            LongConsumer doChallenge)
+        {
+            final long challengeAfter = this.expiresAtMillis - this.challengeResponseTimeoutMillis;
+            if (now >= challengeAfter && now < expiresAtMillis)
```

Comment:
Let's rearrange this to feel like a `between` operator.

`if (challengeAfter <= now && now < expiresAtMillis)`